lib/slim_lint/linter/README.md

```diff
@@ -161,6 +162,22 @@ AllCops:
   DisplayCopNames: true
 ```
 
+## Tab
+
+Reports detection of tabs used for indentation.
+
+**Bad**
+```slim
+.container
+	p Hello World
+```
+
+**Good**
+```slim
+.container
+  p Hello World
+```
+
```

Comment:
This was sort of awkward to illustrate in the readme since it looks the same either way. I'm open to suggestions here.